src/vivarium_gates_child_iv_iron/tools/cli.py

```diff
@@ -41,6 +41,10 @@ def make_artifacts(location: str, output_dir: str, append: bool, replace_keys: T
 
 @click.command()
 @click.argument('output_file', type=click.Path(exists=True))
+@click.option('--groupby-seeds',
```

Comment:
We still want to have a single dash option. Isn't "groupby-seeds" the opposite of this behaviour? I think you should call it `--disaggregate-seeds`.